api-admin/src/main/java/bio/terra/pearl/api/admin/service/auth/context/OperatorAuthContext.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.pearl.api.admin.service.auth.context;
+
+import bio.terra.pearl.core.model.admin.AdminUser;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@Getter
+@Setter
+public class OperatorAuthContext {
+  private AdminUser operator;
+
+  public static OperatorAuthContext of(AdminUser operator) {
+    return OperatorAuthContext.builder().operator(operator).build();
+  }
```

Comment:
Should the other auth contexts extend this class? Or perhaps there could be an interface, `OperatorAwareAuthContext`. That way you can do

```
@SuperUserOnly
@EnforcePortalEnvPermission(...)
public static verySecureMethod(PortalEnvAuthContext authContext, ...) {...}
```

Reply:
I think it's safe to have them extend it.  The reason we had to do EnvironmentAware as an interface was to handle the multple permutations of portal/study/env contexts